.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
+build/
```

Comment:
My only remaining question is: does the removal of the `build` folder makes new features no available until we tag & publish a new release? For instance, two months ago, [support for Lambda was added](https://github.com/grafana/k6-jslib-aws/pull/76), but no new release have been shipped since then.

So, in such case, I guess anyone could import directly from GitHub (does that work? is that correct?), but if we remove `build` directory, that will no longer be possible. Is that something we want? Or we're just fine with no one being able to use new features until they're shipped through a proper release?

Thanks!

Reply:
:thinking: TBH, I assume that importing the library directly from GitHub won't work, and to get the library, you need to somehow _deliver_ it to your script. Either with `jslib.k6.io` or implement some process of cloning (or maybe importing with npm package if it's possible directly with github). 

On the other side, if a customer implements delivery of the library manually (sources download) and relies on the fact that the build is located in this directory, it could be a breaking change. Still, the case sounds hypothetical, but who knows :shrug: 

>For instance, two months ago, https://github.com/grafana/k6-jslib-aws/pull/76, but no new release have been shipped since then.

It's unavailable for most users via `jslib.k6.io`, but if the customer sets up the process of importing code directly from GitHub, it's available.

TL;DR: since the project already has a `build,` and you're right that somebody could rely on the existence of the `build` directory, it's unsafe just deleting it. But the `release` workflow is still essential and could help us with the automation of distribution builds to the `jslib.k6.io`.

Reply:
> My only remaining question is: does the removal of the build folder makes new features no available until we tag & publish a new release?

For context, this is the reason I kept the `build` folder so far, as it allowed to point users to build files in between releases when a fix was not yet released for instance. My understanding is that one could pick the address of the raw build file from GitHub, import it in their script, and it would work (haven't tried it myself, but I assume that works).

From the development of the library's perspective, I think it was convenient to have that folder around. On the other hand, the gain it brought (not having to download the whole library, and build the files for a given commit yourself while developing), is also counterweight-ed by the noise it adds to PRs by including the build files there (which can be mitigated by a CI step as we discussed, but still :)).

I have no strong opinions either way on this one, thus feel free pick to pick the way you judge best 🤝